contracts/bridge/BridgeTransferERC20.sol

```diff
@@ -53,11 +58,9 @@ contract BridgeTransferERC20 is BridgeTokens, IERC20BridgeReceiver, BridgeTransf
             _tokenAddress,
             _value,
             _requestedNonce,
+            lowerHandleNonce,
```

Comment:
Question:
Why `lowerHandleNonce` value is added to the event log?

Reply:
This event can deliver lowerHandleNonce to sub-bridge in realtime.
Using this value, we can know whether a bridge contract works well or not.